### DISCUSSION TITLE
How many archipelago nations are in the 🗺️?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#854] [Galaxy Trivia] [answered]
**Bonus:** What are the largest and smallest nations by territory?

**Comment by PatilHarshh** [comment]
The Largest and smallest nations by territory can vary depending on how territory is defined .
Largest Nation by Territory : Russia (17.1 million Square kilometers)
Smallest Nation by Territory : Vatican City (0.44 Square kilometers)

@ParanoidUser

**Reply by ParanoidUser** [reply]
Nice try! But neither 🇻🇦 nor 🇷🇺 nations are archipelagos.

**Comment by PatilHarshh** [comment]
Largest Archipelago Nation: Indonesia
Smallest Archipelago Nation: Maldives

@ParanoidUser

**Reply by ParanoidUser** [reply]
Nice try! And how many such nations are there in the world?

**Reply by PatilHarshh** [reply]
5 such Nations @ParanoidUser

**Reply by ParanoidUser** [reply]
**Hint:** Undershoot..

**Comment by ParanoidUser** [comment]
**Hint:** We first need to answer “the Ultimate Question of Life, the Universe, and Everything.”. Then just add https://github.com/ParanoidUser/yolo/discussions/854#discussioncomment-6067664 to it.

**Comment by tshenolo** [comment]
Indonesia - largest
Bocas del Toro in Panama - smallest

**Reply by ParanoidUser** [reply]
Nice try! 😊 The world's third-smallest country is actually an island. Can you guess?

**Comment by peterbaccaro** [comment, marked as answer]
There are **27 archipelago nations** in the world, they are:

- Antigua and Barbuda
- Cape Verde
- Comoros
- Fiji
- Indonesia
- Japan
- Kiribati
- Maldives
- Marshall Islands
- Mauritius
- New Zealand
- Palau
- Papua New Guinea
- Philippines
- Samoa
- Sao Tome and Principe
- Seychelles
- Solomon Islands
- St. Kitts and Nevis
- St. Vincent and the Grenadines
- The Bahamas
- The Federated States of Micronesia
- Tonga
- Trinidad and Tobago
- Tuvalu
- United Kingdom
- Vanuatu

The largest archipelago nation by territory is **Indonesia** which has a total land area of approximately 1,910,931 square kilometres (737,815 square miles).

The smallest archipelago nation by territory is **Tuvalu** which is composed of nine coral atolls and has a total land area of approximately 26 square kilometres (10 square miles).
